CONTRIBUTING.md

```diff
@@ -33,11 +33,11 @@ In another terminal, you can for instance navigate to `examples/next-prisma-star
 > Note: you will want to have `pnpm dev` running in parallel in another terminal
 
 ```bash
-# in project root directory
-pnpm test --watch
```

Comment:
We can just change this to `pnpm test-watch` and `pnpm test-watch --testPathPattern react`

Reply:
Oh I like that!